includes/class-admin.php

```diff
@@ -668,6 +679,27 @@ public static function manage_comments_custom_column( $column, $comment_id ) {
 		}
 	}
 
+	/**
+	 * Add the new ActivityPub comment types to the comment types dropdown.
+	 *
+	 * @param array $types The existing comment types.
+	 *
+	 * @return array The extended comment types.
+	 */
+	public static function comment_types_dropdown( $types ) {
+		global $activitypub_comment_types;
+
+		if ( ! is_array( $activitypub_comment_types ) ) {
```

Comment:
What are circumstances where it wouldn't be an array?

Reply:
Vorsicht ist die Mutter der Porzellankiste!

Because it is a global, it can be changed everywhere without any validation. But this might be the wrong place to check it. I think you are right with using the Comment functions and it might make sense to add the check there.

Reply:
It's a global we set. If someone or something manipulates it to change its type, that should be so rare that we shouldn't have to account for it. Especially with good API functions for it where it wouldn't need to be manipulated directly.